tests/gui/TestGui.cpp

```diff
@@ -1280,6 +1280,94 @@ void TestGui::testDragAndDropKdbxFiles()
     QTRY_COMPARE(m_tabWidget->count(), openedDatabasesCount);
 }
 
+void TestGui::testSortGroups()
```

Comment:
Need to test with groups that consist of all numbers (eg, "000", "010", "430") and mixed alphanumeric (eg, "0Group", "Hi50", "Hi300")

Reply:
Agreed. I added these tests to `TestGroup.cpp`.